README.md

```diff
@@ -10,9 +10,9 @@ Writen by Vitor Balocco ([@vitorbal](https://github.com/vitorbal)) and Adam Terl
 ## Usage
 
 ```javascript
-import { createBBStore, combineBBEntities } from 'simple-backbone-redux';
+import { bbCreateStore, bbCombineEntities } from 'simple-backbone-redux';
 
-const bbStore = createBBStore(combineBBEntities({
+const bbStore = bbCreateStore(bbCombineEntities({
```

Comment:
we don't need to call `bbCombineEntities` here anymore, I think? doesn't `bbCreateStore` do that for us now?